src/predictor/gpu_predictor.cu

```diff
@@ -468,37 +467,37 @@ class GPUPredictor : public xgboost::Predictor {
                        std::vector<bst_float>* out_preds,
                        const gbm::GBTreeModel& model, unsigned ntree_limit,
                        unsigned root_index) override {
-    cpu_predictor_->PredictInstance(inst, out_preds, model, root_index);
+    LOG(FATAL) << "Internal error: " << __FUNCTION__
```

Comment:
In https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2019, we see both `__FUNCTION__` and `__func__`.

Reply:
@hcho3 Since `__func__` is defined in the standard, I changed `__FUNCTION__` to it.  Would you like to give a review?  Most of the changes are trivial.

Reply:
I'll take a look.